stemgen.py

```diff
@@ -47,13 +48,11 @@
     "-o",
     "--output",
     dest="OUTPUT_PATH",
-    default="output"
-    if str(INSTALL_DIR) == PROCESS_DIR or INSTALL_DIR.as_posix() == PROCESS_DIR
-    else ".",
+    default="output" if INSTALL_DIR.as_posix() == PROCESS_DIR else ".",
```

Comment:
Can you revert this change please? We need this additional condition on Windows

Reply:
Yep! Sorry for the delay, got held up at work.

Reply:
> every variable would point to your current directory

Only when the `--output` argument is not specified. This is only affecting the default value.